src/components/Home.js

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { FiChevronLeft } from 'react-icons/fi';
+import { HiMicrophone } from 'react-icons/hi';
+import { MdSettings } from 'react-icons/md';
+import { BsArrowRightCircle } from 'react-icons/bs';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchCountries } from '../redux/countries/countries';
+import europe from '../assets/maps/europe.svg';
+import covid from '../assets/icons/virus.svg';
+import '../css/Home.css';
+
+const Home = () => {
```

Comment:
- [ ] As specified, kindly implement a **redux filter/ search** functionality so as to enable filtering of countries. The filter/ search field should be displayed on the Homepage.

Reply:
This now is done.

Reply:
This change has been implemented well